commands/Fun/choice.js

```diff
@@ -0,0 +1,26 @@
+exports.run = (client, msg, choices) => {

```

Comment:
`exports.run = (client, msg, ... [choices]) => {`

Array of choices allowing for multiple args? maybe